.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -111,123 +122,36 @@ The [FINOS Technical Oversight Committee](https://github.com/finos/technical-ove
 If needed, the TOC will request a follow up either via GitHub Issue comments or by inviting project leads to one of their recurrent meetings.
 
 Tasks (for FINOS Infra/TOC)
-- [ ] Prioritise this issue on the [TOC Backlog](https://github.com/orgs/finos/projects/39)
-- [ ] Assign this issue to @eddie-knight
-- [ ] Add the `ready-for-tsc` label
 - [ ] TOC to invite contributors to present their project
 - [ ] FINOS TOC approves/rejects the contribution
-- [ ] Ask @jgavronsky to mark contribution as "Engaged" within LF systems
-- [ ] (optional) If additional socialization is required, the Executive Director may bring projects to the FINOS Governing Board
-- [ ] Update the contribution status to "Engaged" by sending another email to **LF Legal Representative** with the name of the project and its new status.
```

Comment:
Removed and moved to internal Asana task. Also the contribution is updated on PCC not via email (need clarity on how to do this as we cannot edit projects currently, in general we need some clarity on how the LFX team want us to manage this).